app/javascript/spec/host-initiator-group-form/host-initiator-group.spec.js

```diff
@@ -0,0 +1,131 @@
+import React, {useState, useEffect} from 'react';
+
+import { mount } from '../helpers/mountForm';
+
+import toJson from 'enzyme-to-json';
+import {act} from 'react-dom/test-utils';
+
+require('../helpers/miqSparkle.js');
+require('../helpers/miqAjaxButton.js');
```

Comment:
@gregoryb-at-ibm I don't think we need any miqAjaxButton references here as we not using that in react file.